src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2447,6 +2451,8 @@ void os::Linux::print_uptime_info(outputStream* st) {
   int ret = sysinfo(&sinfo);
   if (ret == 0) {
     os::print_dhm(st, "OS uptime:", (long) sinfo.uptime);
+  } else {
+    st->print_cr("OS uptime could not be retrieved.");
```

Comment:
I think an assert would be preferable here. This code can't be parsed so I think it better not to inject it.